tasks/mysql_secure_installation.yml

```diff
@@ -0,0 +1,63 @@
+---
+
+# supported for ansible ver => 2.0
+#- name: Install python-mysqldb for Ansible
+#  package: pkg=python-mysqldb state=present
+
+
+- name: Install python-mysqldb for Ansible
+  apt: name=python-mysqldb state=present
+  when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
+
+- name: Install python-mysqldb for Ansible
+  yum: name=MySQL-python state=present
+  when: ansible_os_family == 'RedHat' or ansible_os_family == 'Oracle Linux'
+
+- name: root password is present
+  mysql_user: name=root host={{ item }} password={{ mysql_root_password }} state=present
+  with_items:
+    - '::1'
+    - '127.0.0.1'
+    - 'localhost'
+
+- name: root has .my.cnf
+  template: src=my.cnf.j2 dest=/root/.my.cnf 
```

Comment:
> There's a problem with dest=/root/.my.cnf. If the user running the role is not root but e.g. becomes root with sudo, login will fail.

This is not true for me as well. I'm using global _become: yes_ in my playbook, connecting to instances as normal user. And _{{ ansible_env.HOME}}_ is  _/root_. I suppose _python-mysqldb_ module uses credentials from user's home directory. In this case it is /root, the same path as ansible_env.HOME. So I can set a variable: `mysql_user_home: "{{ ansible_env.HOME}}"`, but if user uses become, it changes nothing. And if user doesn't use become - a lot of things will fail much earlier =)